pkg/cfn/manager/api.go

```diff
@@ -450,15 +450,22 @@ func (c *StackCollection) DescribeStacks() ([]*Stack, error) {
 	return stacks, nil
 }
 
-func IsClusterStack(stacks []*Stack) bool {
+func (c *StackCollection) IsClusterStack() (bool, error) {
+	clusterStackName := c.makeClusterStackName()
+	stacks, err := c.DescribeStacks()
```

Comment:
does this definitely hit the API again?

Reply:
It does. BTW, this is a clone PR for https://github.com/weaveworks/eksctl/pull/3057